sound/soc/sof/intel/hda.c

```diff
@@ -966,8 +967,9 @@ static int hda_init_caps(struct snd_sof_dev *sdev)
 	 */
 	ret = hda_sdw_probe(sdev);
 	if (ret < 0) {
-		dev_err(sdev->dev, "error: SoundWire probe error\n");
-		return ret;
+		dev_err(sdev->dev, "SoundWire probe error: %d\n", ret);
+		i915_off = true;
```

Comment:
Why `i915_off = true` is only set here? When will i915 be turned on? Do we need to set `i915_off = true` in every `return` and `goto` above?

Reply:
dropped